linux-tools/pexpect_test/pexpect_test.py

```diff
@@ -30,6 +30,10 @@ def run_once(self, test_path=''):
         """
         try:
             os.environ["LTPBIN"] = "%s/shared" %(test_path)
+            cwd = os.getcwd()
+            os.chdir("%s/pexpect_test" %(test_path))
+            os.system("patch -p0 < pexpect-pxssh-scripts.diff")
```

Comment:
let us have a check for return value and proceed it if patch applied successfully